Assets/Scripts/PlayerMovement.cs

```diff
@@ -17,9 +19,12 @@ void Update()
     {
         moveInput = Input.GetAxis("Horizontal"); // if I press right key moveInput = 1,left key = -1 
         ballRigidBody.velocity = new Vector2(moveInput * speed, ballRigidBody.velocity.y); // adds force to the ball movement in order for the player to move,doesn't affect y axis
-        if (Input.GetKeyDown(KeyCode.Space) && grounded) // checks if jumping key is pressed
+        if (grounded)
+            remainingJumps = maxJumps; // if the player is grounded the remaningJumps variable is equal to the maxJum
+        if (Input.GetKeyDown(KeyCode.Space) && remainingJumps > 0) // checks if jumping key is pressed
```

Comment:
write `  if (Input.GetKeyDown(KeyCode.Space) && remainingJumps > 0 && grounded) // checks if jumping key is pressed
`
that way you have one layer of protection for the ball and you make sure that the ball can only jump when it is on the ground. the line translates to "if you press space and you have remaining jumps left and if the ball is on the ground"